### PR TITLE
Update rules

### PR DESCRIPTION
**Pull request by lukefrake**

### CODE REVIEW

**Comment by lukefrake** [comment]
mixins-before-declarations - Not something we enforce
no-qualifying-elements - Doesn't work correctly
class-name-format - Using multi types of BEM
nesting-depth - Need over 2 when using modernizr

**Comment by MCooke** [comment]
+1

File: configs/.sass-lint.yml [hunk 1]
```diff
@@ -40,10 +40,7 @@ rules:
   force-pseudo-nesting: 0
 
   # Name Formats
-  class-name-format:
-    - 1
-    -
-      convention: hyphenatedbem
+  class-name-format: 0
```
Comment (f000) [inline configs/.sass-lint.yml:43]: bem or hyphenated_lowercase would be better

File: configs/.sass-lint.yml [hunk 2]
```diff
@@ -5,7 +5,7 @@ rules:
   placeholder-in-extend: 0
 
   # Mixins
-  mixins-before-declarations: 1
+  mixins-before-declarations: 0
```
Comment (f000) [inline configs/.sass-lint.yml:8]: 👎

File: configs/.sass-lint.yml (hunk 1 above)
Comment (lukefrake) [inline configs/.sass-lint.yml:43]: We are currently using BEM with single and double hyphens. We need to agree the standard then we can put this back in

File: configs/.sass-lint.yml (hunk 2 above)
Comment (lukefrake) [inline configs/.sass-lint.yml:8]: This doesn't work too good with the media query mixins.

Meant lots of changes for the repo I was putting this into so once we agree a standard we can put it back in.

Comment (f000) [inline configs/.sass-lint.yml:8]: ok np

**Pull request merged**